storybook/storybook-react/src/Spotlight/Spotlight.stories.tsx

```diff
@@ -23,17 +23,17 @@ const meta = {
       control: {
         type: 'select',
         labels: {
-          blue: 'Blauw',
-          'dark-green': 'Donkergroen',
-          green: 'Groen',
-          'light-blue': 'Lichtblauw',
+          blue: 'Blue',
```

Comment:
In the Badge stories these labels are kebab-cased. Do we want to change those to align with these?

Reply:
We discussed removing translations of prop values altogether.